source/frequently-asked-questions/frequently-asked-questions.rst

```diff
@@ -2,3 +2,21 @@
 Frequently Asked Questions
 ################################################################################
 
+* **How can I store big files on the blockchain?**
+  Swarm_ is an Ethereum-specific project for distributed file storage. IPFS_ is an independent project which has close ties to Ethereum; it will be used independently and may be used as the layer underlying Swarm.
+* **Is ethereum based on bitcoin?**
+  Only in the sense that it uses a blockchain, which Bitcoin pioneered. Ethereum has a separate blockchain that has several signigicant technical differences from Bitcoin. 
+* **What other cool apps are being built?** See this list from Ethercasts_.
```

Comment:
there is a section on Dapps where all dapps listings will appear not just this one.